src/applications/edu-benefits/0994/utils.jsx

```diff
@@ -13,3 +13,22 @@ export const srSubstitute = (srIgnored, substitutionText) => (
     <span className="sr-only">{substitutionText}</span>
   </span>
 );
+
+export const maskBankInformation = (string, unmaskedLength) => {
```

Comment:
Might be useful to have unit tests for this

Reply:
I have added an unit test